README.md

```diff
@@ -21,62 +22,102 @@
 
-- ES6 Compatibility
-  - How to handle...
-  - let/const/var?
-  - async/await? 
-  - getters/setters? 
-  - static properties, methods, etc?
-  - for in/for of?
-  - decorators (and their possible clash with @properties)?
-- Should types (eg; via Flow) be supported? If so, how?
-- What should the compiler look like? 
-- What level of backwards-compatibility with CoffeeScript should be targeted?
-- Should new features be added? Which ones? What should the bar for new features be?
-- What are the biggest warts in CoffeeScript that ought to be removed or patched?
+CoffeeScript is hemorrhaging marketshare the longer that incompatibilities with popular frameworks and build tools go unaddressed. We must support modules and classes ASAP, and that means building them in the current compiler. In the case of classes, that probably means a flag to opt-in to new ES2015 class syntax and output. Any other missing ES2015 features that preclude CoffeeScript’s selection for a project must also be addressed immediately, as patches to the current compiler.
 
-... and surely many more. Kindly [open an issue](https://github.com/coffeescript6/discuss/issues/new) to open a discussion on any of the above, or similar, topics!
+### Developer Happiness: Supporting Optional ES2015+ Features
 
+Features that developers enjoy in ES2015 that are appropriate to implement in CoffeeScript, like `const`, should be implemented as time permits. These features should be implemented in a new compiler, assuming we build one; and possibly also in the old compiler too if such an implementation is easy and doesn’t break backward compatibility. Any feature implemented in both compilers should have identical syntax.
 
-## The name
-Given that this project has no official blessing from @jashkenas and co – and may never receive it – the name "CoffeeScript 6.0" should be thought of as a placeholder. 
+### Modernizing Output
 
+Lastly, CoffeeScript’s output should be modernized as much as possible. Fat arrows should output as fat arrows, destructuring should output as destructuring, etc. But this should be the **last priority**—it makes our source code cleaner, and debugging eventually easier when ES2015 features are supported natively in runtimes, but otherwise ES2015 output has little benefit for developers. That said, *new* features, especially ES2015+ ones, should definitely be output as ES2015+ code.
 
-## Possible Alternatives
-There are a number of interesting new languages that have sprung up, some of them listed below. None seem to capture the essence of "unfancy JavaScript" accomplished in CoffeeScript, but may be sources of inspiration nonetheless:
+## A New Compiler
+
+The current CoffeeScript 1.x compiler is extremely well commented, but it really needs those comments. Understanding it will bend your brain into knots. The original goal of [CoffeeScript Redux](https://github.com/michaelficarra/CoffeeScriptRedux) was to refactor CoffeeScript’s codebase, not to add new features.
```

Comment:
Good point, I’ll revise this the next time I edit the document. Or you can submit a PR 😉